src/http.ts

```diff
@@ -197,10 +240,9 @@ export class LemmyHttp {
 
   /**
    * Gets the site, and your user data.
-   *
-   * `HTTP.GET /site`
```

Comment:
I removed all these `HTTP.GET + route` in the descriptions, because this is inherent in the OpenAPI docs now.